src/parseThresholds.test.ts

```diff
@@ -1,25 +1,25 @@
 import { describe, it, expect } from 'vitest';
 import path from 'path';
-import { parseThresholds } from './parseThresholds';
+import { parseCoverageThresholds } from './parseCoverageThresholds';
```

Comment:
I like the renaming of the function - but this import now does not match the filename anymore so the test fails. 🤭 

However, maybe just rename both files (so `parseThresholds.ts` -> `parseCoverageThresholds` and `parseThresholds.test.ts` -> `parseCoverageThresholds.test.ts`) so they align with the function name.

Reply:
Gah, I cannot believe I did something that stupid (but I guess semi-understandable as I could not test) 🤦...

Reply:
That is not dumb, things happen. Development is messy, you would be the first developer not to introduce bugs - so all good! 🙂